src/app/lib/utils.tsx

```diff
@@ -1,4 +1,5 @@
 import { faker } from '@faker-js/faker';
+import axios from 'axios';
```

Comment:
@Lombardoc4 is there any particular reason to use axios over fetch?

Reply:
A little more comfortability in the moment. Going to refactor this away along with faker in next rev to start to properly handle server errors.